apps/gallery/js/gallery.js

```diff
@@ -421,8 +428,15 @@ function setView(view) {
       }
       break;
     case LAYOUT_MODE.edit:
+      $('edit-header').removeAttribute('no-font-fit');
       NFC.unshare();
       break;
+    case LAYOUT_MODE.pick:
+      $('pick-header').removeAttribute('no-font-fit');
+      break;
+    case LAYOUT_MODE.crop:
+      $('crop-top').removeAttribute('no-font-fit');
+      break;
```

Comment:
I think I would slightly prefer the simplicity of calling removeAttribute for all of these headers in startup.js, after the performance.mark('visuallyLoaded') call.

Could you just do something like this in setup.js:

  document.querySelectorAll('gaia-header').forEach(function(header) { header.removeAttribute('no-font-fit'); });

That would be slightly less efficient than this code, but simpler and easier to maintain, and it keeps the font-fit off the startup path.

If you've got good reasons to put this code here in setView(), let's discuss them in the bug. But setView() is already more complicated than I'd like.